SETUP/UNICODE.md

```diff
@@ -0,0 +1,102 @@
+# Unicode support
+
+The DP codebase fully supports UTF-8, both in the HTML rendered to the user
+as well as in page texts.
```

Comment:
Grammar police comment: you don't need "both" and "as well as" -- I recommend either `, ... as well as ...` or `both ... and ...`.

Reply:
Fixed.